### PR TITLE
docs: Build docs on Travis

### PR DESCRIPTION
**Pull request by tonyo**
...so they can later be published by "craft".

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,6 +15,7 @@ pip-log.txt
 .idea
 .eggs
 venv
+.venv
```
Comment (untitaker) [inline .gitignore]: hmm, where does this venv come from

Comment (tonyo) [reply .gitignore]: It was created when I ran `make test`

Comment (untitaker) [reply .gitignore]: oh, ok 👍 just saw that the entire .gitignore is preemptive anyway

File: .travis.yml [hunk 2]
```diff
@@ -28,14 +28,20 @@ matrix:
...
+        - pip install tox
       script: tox -e linters
     - python: "3.6"
-      env: DIST=1
-      script: make dist
-      after_success:
-        - npm install -g @zeus-ci/cli
-        - zeus upload -t "application/zip+wheel" dist/*
+      name: Distribution packages
+      install: false
```
Comment (tonyo) [inline .travis.yml]: A bit of optimization: I don't think we need Rust and tox for this job.

Comment (untitaker) [reply .travis.yml]: :+1:

**Comment by tonyo** [comment]
@untitaker Ready for review 📜

**Pull request merged**